### DISCUSSION TITLE
alerts.weather.gov

### DISCUSSION THREAD

**Discussion by tmastersmart** [weather-gov/api#642] [Q&A]
alerts.weather.gov is still providing 1.1 data where is the 1.2 data I can find nothing in api.weather.gov that will produce alerts
right now im getting special weather statements on alerts.weather.gov but nothing at all on the api.
is alerts.weather.gov going to be shutdown or will it convert to the new format?

"NWS CAP v1.1 is being terminated. NWS CAP users should instead use CAP v1.2. Please see [Service Change Notice (SCN) 17-35](https://www.weather.gov/media/notification/pdfs/scn17-35capaaa.pdf) for"
the above doc does not exist

**Comment by dwhitemv25** [comment]
The current SCN for the CAP v1.1 to v1.2 migration is here:

https://www.weather.gov/media/notification/pdf_2023_24/scn23-57_termination_cap_v1.1.pdf

It does reference SCN 17-35, but the link is dead. That said, the XML CAP v1.2 format alerts are available through the API, just set your `Accept:` header to `application/atom+xml` for lists of alerts and `application/cap+xml` for individual alerts by ID. 

I don't see any notifications about changes to alerts.weather.gov, so I suspect it isn't going away anytime soon, but isn't likely to see any more updates. 

I can confirm that Special Weather Statements (SPS) are showing up as an active alert in the API. At this moment there is one open in zone WAZ504 and it shows up on the API with this query:

https://api.weather.gov/alerts/active/zone/WAZ504

Documentation for the API is here:

https://www.weather.gov/documentation/services-web-api 

Can you provide examples of queries that are not providing the information you're looking for?
